zenodo/modules/records/serializers/to_marc21/rules.py

```diff
@@ -107,3 +107,18 @@ def reverse_conkerence_url(dummy_self, key, value):
         '$ind1': '4',
         '$ind2': '_',
     }
+
+
+@to_marc21.over('856', '^_files$')
+@utils.reverse_for_each_value
+@utils.filter_values
+def reverse_files(dummy_self, key, value):
+    """Reverse - Files."""
+    return {
+        's': str(value['size']),
+        'u': value['uri'],
+        'w': value['checksum'],
```

Comment:
Yes, I misinterpreted it - I'll remove the $w.
Should the checksum be in $z perhaps? I think checksum is more important to include than ordering?